src/main/webapp/election-info.js

```diff
@@ -256,4 +256,18 @@ function addPollingStationMarker(map, position, title, description) {
   marker.addListener('click', () => {
     infoWindow.open(map, marker);
   });
-}
\ No newline at end of file
+}
+
+/**
+ * Handlebars helper that removes the beginning "http://" and the trailing "/"
```

Comment:
I believe this will also catch "https://". You should comment that as well to be more precise. 

BTW to clarify regarding comments, usually "why" not "what" is the rule, but function docs are a bit of an exception to that, since there's often a lot of subtleties to clarify in functions. Regex is also an exception because nobody can read regex easily lol.

Reply:
done